README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
+
+## Install
+```python
+$ pip install things.py
+```
+
+## Run
+```pycon
+>>> import things
+>>> things.tasks()
```

Comment:
Might need something like
```python
things = Things()
things.tasks() // or things.get_tasks()
```
See e.g. https://github.com/thingsapi/things.py/blob/0e3f3b3389383ca4dd3604f0d6d1769148c9048d/tests/test_things.py#L14

Reply:
I don't know about you, but I like the approach of the requests library here.

`requests.get(...)` is a shorthand for

```
    with sessions.Session() as session:
        return session.request(method=method, url=url, **kwargs)
```

Similarly, we could have

```things.tasks(database='tests/demo.sqlite3')``` be a shorthand for

```
    with Session(database='...') as session:
        return session.tasks(...)
```

Reply:
pandas does a similar approach to avoid having to import and instantiate classes explicitly.

Reply:
lgtm